packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -514,3 +544,203 @@ contract StandardValidatorV180 {
         return keccak256(bytes(_a)) == keccak256(bytes(_b));
     }
 }
+
+contract StandardValidatorV180 is StandardValidatorBase {
+    struct InputV180 {
+        IProxyAdmin proxyAdmin;
+        ISystemConfig sysCfg;
+        bytes32 absolutePrestate;
+        uint256 l2ChainID;
+    }
+
+    constructor(
+        ImplementationsBase memory _implementations,
+        ISuperchainConfig _superchainConfig,
+        address _l1PAOMultisig,
+        address _mips,
+        address _challenger
+    )
+        StandardValidatorBase(_implementations, _superchainConfig, _l1PAOMultisig, _mips, _challenger)
+    { }
+
+    function validate(InputV180 memory _input, bool _allowFailure) public view returns (string memory) {
+        string memory _errors = "";
+
+        _errors = super.validate(_errors, _input.sysCfg, _input.proxyAdmin, _input.absolutePrestate, _input.l2ChainID);
+
+        if (bytes(_errors).length > 0 && !_allowFailure) {
+            revert(string.concat("StandardValidatorV180: ", _errors));
+        }
+
+        return _errors;
+    }
+}
+
+contract StandardValidatorV200 is StandardValidatorBase {
+    address public superchainConfigImpl;
+    address public protocolVersionsImpl;
```

Comment:
Why don't we declare these vars in `StandardValidatorBase` and validate these contracts in `StandardValidatorV180` also?

Reply:
They're not used in the v180 OPCM so I separated them. Happy to move all that validation inside the base contract.